### PR TITLE
Displaying cohort opt-out confirmation before saving the choice

### PR DESCRIPTION
**Pull request by BroiSatse**
https://dfedigital.atlassian.net/jira/software/projects/CPDRP/boards/102?selectedIssue=CPDRP-446

### CODE REVIEW

File: app/forms/induction_choice_form.rb [hunk 1]
```diff
@@ -21,6 +21,10 @@ def programme_choices
     ]
   end
 
+  def programme_choice_words
+    programme_choices.find { |opt| opt.id == programme_choice }&.name
```
Comment (kicferk1) [inline app/forms/induction_choice_form.rb:25]: Let's use full words, `option` > `opt`

File: app/views/schools/choose_programme/confirm_programme.html.erb [hunk 2]
```diff
@@ -9,11 +9,7 @@
     <p class="govuk-body">You have chosen to:</p>
 
     <ul class="govuk-list govuk-list--bullet">
-      <% if @induction_choice_form.programme_choice == "full_induction_programme" %>
-        <li>Use a training provider, funded by the DfE</li>
-      <% else %>
-        <li>Deliver your own programme using DfE accredited materials</li>
-      <% end %>
+      <li><%= @induction_choice_form.programme_choice_words %></li>
```
Comment (kicferk1) [inline app/views/schools/choose_programme/confirm_programme.html.erb:12]: Is this the same? I think the programme_choice have `(full induction programme)` and `(core induction programme)` in them in addition to those options.

Comment (kicferk1) [reply app/views/schools/choose_programme/confirm_programme.html.erb:12]: And percy noticed that :O 
Also, looks like cypress tests are broken 😢

Comment (BroiSatse) [reply app/views/schools/choose_programme/confirm_programme.html.erb:12]: Yup, there are small changes. I was hoping those are all right given that FIP and CIP are a technical jargon. Content files are not too helpfull there (I don't know what does it mean when the text in brackets is stroke orange?) and prototype doesn't show these neither.

Cypress fixed.

**Review by kicferk1 — APPROVED** [review]

**Comment by kicferk1** [comment]
One of the new pages says 
> You have chosen to
> We don't expect new teachers...

Not sure if it is the right wording :P

**Review by kicferk1 — APPROVED** [review]

**Pull request merged**